client/my-sites/post-selector/pagination.jsx

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
>I mean removing the PostSelectorPagination child and making QueryPosts + Selector direct children of PostSelector.

Yeah, in retrospect, that should be possible, and would help eliminate some of the redundancy in `<PostSelectorPagination />` with passing props to the selector. I'll see to tidying this up.